kalite/distributed/api_views.py

```diff
@@ -82,9 +84,25 @@ def compute_total_points(user):
         return None
     else:
         return (VideoLog.get_points_for_user(user) +
-           ExerciseLog.get_points_for_user(user) +
-           StoreTransactionLog.get_points_for_user(user) +
-           ContentLog.get_points_for_user(user))
+                ExerciseLog.get_points_for_user(user) +
```

Comment:
I'm unclear... are you saying we should prefer a single tabwidth, instead of aligning with the start of the first variable name?